Source/player.h

```diff
@@ -16,6 +16,7 @@
 #include "path.h"
 #include "interfac.h"
 #include "utils/enum_traits.h"
+#include "animationinfos.h"

```

Comment:
We do, clang-tidy will apply the rule but haven't applied the other rules. I suggest we just ignore it for now as it will be corrected automatically anyway.